cpp/src/c_api/capi_helper.cu

```diff
@@ -74,6 +75,28 @@ template void sort_by_key(raft::handle_t const& handle,
                           raft::device_span<int64_t> keys,
                           raft::device_span<int64_t> values);
 
+template <typename T>
+rmm::device_uvector<T> device_allgatherv(raft::handle_t const& handle,
+                                         raft::device_span<T const> d_input)
+{
+  auto& comm      = handle.get_comms();
```

Comment:
I'd be inclined to either:
 1. Consolidate this to one line (`return cugraph::device_allgatherv(handle, handle.get_comms(), ...)`), or
 2. Add the comm as a parameter to match the called method.

Leaning toward the latter (match the function signatures match).  Thoughts @seunghwak ?

Reply:
Got it I chose the the latter.